### DISCUSSION TITLE
How to show Content Type name in search refiners without file type prefix?

### DISCUSSION THREAD

**Discussion by LetMeSP** [microsoft-search/pnp-modern-search#1490] [Q&A] [answered]
I am using a ContentType option in the Filter Field for a refiner. It shows correct content types but with the file type prefix (see screenshots). What do I need to do to only show the Content Type name in the refiner panel?

![2021-11-05_11-47-14](https://user-images.githubusercontent.com/41652178/140563392-c5525a7c-6a8c-4101-b632-25c4cdffec6b.png)

![2021-11-05_11-47-52](https://user-images.githubusercontent.com/41652178/140563402-43013906-bbbe-4c79-bb1a-9bada219ed0e.png)

**Comment by wobba** [comment, marked as answer]
Never use `ContentType` as a filter/managed property, use `SPContentType`. One of those old tricks which are hard to figure out :)
